Libraries/Components/Touchable/Touchable.js

```diff
@@ -668,9 +668,10 @@ var TouchableMixin = {
       this.touchableHandleActivePressIn && this.touchableHandleActivePressIn(e);
     } else if (!newIsHighlight && curIsHighlight && this.touchableHandleActivePressOut) {
       if (this.touchableGetPressOutDelayMS && this.touchableGetPressOutDelayMS()) {
-        this.pressOutDelayTimeout = this.setTimeout(function() {
```

Comment:
@ide updated... Was going to initially, saw other binds in the file doing the same thing so went with that syntax.